### ISSUE TITLE
Using the electron package in AUR

### ISSUE THREAD

**Issue by j605** [techwiz24/aur-ghetto-skype#1]
I don't really have a solution but can you look into the electron package and other packages that use it. I don't want to download electron for every single application I use.

**Comment by nlowe** [comment]
So I think we can pull this off, but we have to change

``` js
const BrowserWindow = require('browser-window');
```

to

``` js
const BrowserWindow = require('electron').BrowserWindow;
```

in `main.js`.

I'll have a new pkgbuild shortly

**Comment by nlowe** [comment]
@j605 I've published a new copy to the AUR with commit 82454bf1e6d9ebcf7ba6abd47c9cde902f63b011. I've done some basic testing and that seems sufficient. Let me know if that works for you.
